### ISSUE TITLE
High DPI Issues

### ISSUE THREAD

**Issue by RobertBColton** [enigma-dev/lgmplugin#60]
This issue is carried over from ENIGMA where I originally posted it (https://github.com/enigma-dev/enigma-dev/issues/970). This is in fact an issue with the plugin because it is by default rendering fonts at the DPI of my screen, whereas GMS1.4 always renders the fonts at the same exact DPI (probably 96 or 72).

https://github.com/enigma-dev/lgmplugin/blob/ccc750ac54745d05599299549c13a29d91b659d4/org/enigma/EnigmaWriter.java#L663

I thought about this issue some (https://github.com/enigma-dev/enigma-dev/issues/970#issuecomment-388521776) and decided that what GM does here is correct. The font rendering should be independent of the screen size of your IDE or where you are building the game. The fonts should be consistently rendered at the same DPI, and the user should make multiple fonts if they want DPI scaling in their game. Further, it may be helpful to add a per-project setting for the font rendering DPI.

Long story short, remove the call to obtain the current DPI and just use 96 always and the plugin is fixed.

**Comment by RobertBColton** [comment]
Closing as resolved by #970
